crates/gitbutler-hunk-dependency/src/hunk.rs

```diff
@@ -14,6 +16,11 @@ pub struct HunkRange {
 impl HunkRange {
     pub fn intersects(&self, start: u32, lines: u32) -> bool {
         if self.lines == 0 {
+            if self.start == 0 && self.line_shift <= 0 {
+                // File completely removed
+                return true;
+            }
+            println!("line shift: {}", self.line_shift);
```

Comment:
If printing is desired and it shouldn't be `dbg!()` for formatting reasons, one should always use `eprintln()` so the output goes to stderr (unbuffered), which is typically more instantaneous.

Reply:
@Byron Oh that shouldn't be there at all!
Thanks for pointing it out :)